.colab/creating_reusable_components/reusing_previous_work.ipynb

```diff
@@ -0,0 +1,913 @@
+    "id": "4a1bScthQFMl"
+   },
+   "source": [
+    "# Sloan Digital Sky Survey Classification\n",
+    "## Classification of Galaxies, Stars and Quasars based on the RD14 from the SDSS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "b9d4b609-9f0f-4296-8462-98975b2ece09",
+    "_uuid": "16d228ddb3b0d71d6e13093552c04a21146b75e5",
+    "id": "IeFWMBAZQFMo"
+   },
+   "source": [
+    "### Importing Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "_cell_guid": "13914766-c2fb-4801-8846-6c78e6d1cb03",
+    "_uuid": "5bb212bdb34abc34f8bed1a0bc2d1a6287166221",
+    "id": "Yz8i8KUNQFMp"
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import style\n",
+    "import seaborn as sns\n",
+    "sns.set_style('whitegrid')\n",
+    "from sklearn.model_selection import train_test_split, cross_val_predict\n",
+    "from sklearn.svm import SVC\n",
+    "from sklearn.ensemble import RandomForestClassifier\n",
+    "from sklearn.linear_model import SGDClassifier\n",
+    "from sklearn.neighbors import KNeighborsClassifier\n",
+    "from sklearn.naive_bayes import GaussianNB\n",
+    "from sklearn.decomposition import PCA\n",
+    "from sklearn.preprocessing import LabelEncoder, MinMaxScaler\n",
+    "from sklearn.metrics import confusion_matrix, precision_score, recall_score, f1_score\n",
+    "import time\n",
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\", category=DeprecationWarning)\n",
+    "%matplotlib inline\n",
+    "\n",
+    "SMALL_SIZE = 10\n",
+    "MEDIUM_SIZE = 12\n",
+    "\n",
+    "plt.rc('font', size=SMALL_SIZE)\n",
+    "plt.rc('axes', titlesize=MEDIUM_SIZE)\n",
+    "plt.rc('axes', labelsize=MEDIUM_SIZE)\n",
+    "plt.rcParams['figure.dpi']=150"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "_cell_guid": "ffb06ef6-73f7-4f42-ab42-5d5b5f773ba7",
+    "_uuid": "04e88f8c9c12167a1c23e47b3e2046246510e983",
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 336
+    },
+    "id": "-xtqrprxQFMr",
+    "outputId": "5ddf8b35-3410-4f0a-af50-fce5cf843101"
+   },
+   "outputs": [],
+   "source": [
+    "#sdss_df = pd.read_csv('Skyserver_SQL2_27_2018 6_51_39 PM.csv', skiprows=1)\n",
+    "sdss_df = pd.read_csv('data/Skyserver_SQL2_27_2018 6_51_39 PM.csv', skiprows=0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "75a847f1-0dfc-4228-9cbc-49d6034463e5",
+    "_uuid": "9b01bc847e158cfa00d411ea687cb573a0037aef",
+    "id": "hyR_FbSwQFMs"
+   },
+   "source": [
+    "The dataset has 10000 examples, 17 feature columns and 1 target column. 8 of the 17 features are 64 bit integers, 1 feature is an unsigned 64 bit integer, 8 are 64 bit floats and the target column is of the type object. "
```

Comment:
can we give the name of the target column here? its good to know up front in DS what is your "training_data" and what is your "training_labels"

Reply:
in fact lets do a "sdss_df.head()" here because the training label "class" doesn't show up in the describe and is quite confusing.